release/update_and_create_archives.sh

```diff
@@ -59,23 +59,19 @@ done
 [[ -z "${VER_STRING}"   ]] && usage
 
 function copy_and_archive() {
-  # can't seem to set/override PROXY_TAG (sha), FORTIO_HUB ("docker.io/istio"), FORTIO_TAG ("0.3.1")
+  # can't seem to set/override PROXY_HUB, FORTIO_HUB ("docker.io/istio"), FORTIO_TAG ("0.3.1")
   ${ROOT}/install/updateVersion.sh -c "${DOCKER_HUB_TAG}" -A "${DEBIAN_URL}"   -x "${DOCKER_HUB_TAG}" \
                                    -p "${DOCKER_HUB_TAG}" -i "${ISTIOCTL_URL}" -P "${DEBIAN_URL}" \
-                                   -r "${VER_STRING}" -E "${DEBIAN_URL}"
-  # save -d "${OUTPUT_PATH}" for later
+                                   -r "${VER_STRING}" -E "${DEBIAN_URL}" -d "${OUTPUT_PATH}"
   
-  pushd ${ROOT}
-  cp istio.VERSION LICENSE README.md "${OUTPUT_PATH}/"
-  find samples install -type f \( -name "*.yaml" -o -name "cleanup*" -o -name "*.md" \) \
```

Comment:
the mesh expansion scripts need istio.VERSION (unless you changed/fixed that)
and the samples yaml are needed too ? (unless I miss something that also generates those now ?)

Reply:
Prior to this change updateVersion.sh was updating the istio.VERSION in the source tree, but now that -d is used the istio.VERSION is being generated in $OUTPUT_PATH so there's no need to copy the file to this directory.  As for the other content, that's now copied by updateVersions.sh in the block I've pasted below.  This means that more files than before are being copied to OUTPUT_PATH, though these won't get into the tar since create_release_archives.sh also uses "find" to filter out files (I didn't like that update_and_create_archives.sh and create_release_archives.sh both had instances of these "find" blocks [and within the last 1-2 months someone incorrectly added "kubeconfig" to only 1 of these], so I'm using this opportunity to get rid of the "find" block).

if [[ "$DEST_DIR" != "$ROOT" ]]; then
  cp -R $ROOT/install $DEST_DIR/
  cp -R $ROOT/samples $DEST_DIR/
fi

Reply:
Awesome, thanks. lgtm then !
(ps about samples/  and install/ we could probably put in the tar everything except samples/bookinfo/src )